docs/app/documentation/component-docs/button-split/examples/button-split-template-example.component.html

```diff
@@ -0,0 +1,17 @@
+<fd-button-split>
+    <ng-template fd-button-split-action-title>
+        <u>Action Underscored</u>
```

Comment:
Better to make it more obvious that this is for complex action titles. Maybe adding an icon on the left side of the text instead?